tests/python/unittest/test_tir_analysis_get_block_access_region.py

```diff
@@ -0,0 +1,57 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import tvm
+from tvm import tir, script
+from tvm.ir import Range
+
+
+@tvm.script.tir
+def func() -> None:
+    A = tir.alloc_buffer((128, 128), "float32")
+    B = tir.alloc_buffer((128, 128), "float32")
+    C = tir.alloc_buffer((128, 128), "float32")
+    D = tir.alloc_buffer((128, 128), "float32")
+    with tir.block([]):
```

Comment:
Is the empty array (`[]`) required? I think it would be cleaner to default to and empty array. i.e. `tir.block()`.

Reply:
It is required now. usually, a block will not contain an empty iter_var array. We can update it in a separate PR.